pylint/test/extensions/test_check_raise_docs.py

```diff
@@ -115,10 +115,14 @@ def my_func(self):
             """This is a docstring.
 
             :raises RuntimeError: Always
-            :raises NameError: Never
+            :except NameError: Never
```

Comment:
Currently the checker doesn't warn for raises statements in the docstring that don't have an actual exception being raised in the function. In testing to verify this, I did encounter an additional issue:

```python
    def test_ignore_spurious_sphinx_raises(self):
        raise_node = astroid.extract_node('''
        def my_func(self):
            """This is a docstring.

            :raises RuntimeError: Never
            :except NameError: Never
            :raise OSError: Never
            :exception ValueError: Never
            """
            pass
        ''')
        with self.assertNoMessages():
            self.checker.visit_raise(raise_node)
```

causes the following error:
```
Traceback (most recent call last):
  File ".tox/py27/lib/python2.7/site-packages/pylint/test/extensions/test_check_raise_docs.py", line 126, in test_warn_spurious_sphinx_raises
    self.checker.visit_raise(raise_node)
  File ".tox/py27/lib/python2.7/site-packages/pylint/extensions/docparams.py", line 162, in visit_raise
    expected_excs = utils.possible_exc_types(node)
  File ".tox/py27/lib/python2.7/site-packages/pylint/extensions/_check_docs_utils.py", line 61, in possible_exc_types
    if isinstance(node.exc, astroid.Name):
AttributeError: 'FunctionDef' object has no attribute 'exc'
```

I fixed this by adding hasattr() checks in possible_exc_types - will include that in the updated diffs for your review.